chart/hyrax/values.yaml

```diff
@@ -273,6 +411,7 @@ global:
   postgresql:
     postgresqlUsername: hyrax
     postgresqlPassword: hyrax_pass
+  hyraxName: hyrax
```

Comment:
Would it be worth giving this a name that more directly conveys it being used for `nginx` configuration? Or, might it be used elsewhere as well? On initial read I didn't understand why this new value was here, but then found it in the `nginx.serverBlock` setup.

Reply:
I went back and forth on it. on the one hand we only use it once right now. on the other hand, its global and potentially useful in other places. Would it feel better as `.Values.global.hyraxHostName` or `hyraxHostUrl`?

Reply:
@mcritchlow have you had a chance to think more about this? I've been using this code in prod for a week now and it is working well.  We've had this bot blocker set up on another Rails app for over a year and I highly recommend it.

Reply:
no problem on the delay - I 100% understand. I already updated the description to say Cantaloup comes later. I agree with you on the name, I'll push an update.